src/main/java/org/tron/common/runtime/vm/program/InternalTransaction.java

```diff
@@ -105,6 +136,10 @@ public void setTransaction(Transaction transaction) {
     this.transaction = transaction;
   }
 
+  public byte[] getTransferToAddress() {
+    return transferToAddress;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Return a copy of "transferToAddress". ([squid:S2384](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2384))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1589&issues=AWZd7Lql9tx2us2yJnRD&open=AWZd7Lql9tx2us2yJnRD)

[ISSUE_KEY]: AWZd7Lql9tx2us2yJnRD